### PR TITLE
test: unified instances of empty table in all tests

### LINKED ISSUE #347: style: Inconsistency in creating empty tables in tests
### Is your feature request related to a problem?

There's some inconsistency as to how create empty tables in test cases. Most of the time, we're using `Table()`, sometimes we use `Table({})`.

The latter is used in:
* `test_add_column` (2 times)
* `test_add_columns` (2x)
* `test_from_rows.py` (first set of parametrize-testcases)
* `test_remove_columns_with_missing_values` (2x)
* `test_remove_columns_with_non_numerical_values` (2x)
* `test_remove_rows_with_missing_values` (2x)
* `test_repr_html` (3x)
* `test_to_dict`
* `test_to_html`

Do we care about this?

If so, should change these cases to also use `Table()`, so it's the same everywhere.

### Desired solution

Change the above cases to use `Table()`.

### Possible alternatives (optional)

Decide that we don't care.

### PR DESCRIPTION
**Pull request by daniaHu**
Closes #347

### Summary of Changes

Searched through tests for other forms of creating empty table than our agreed on standard "Table()", and switched them to the standard.

### CODE REVIEW

**Review by jxnior01 — CHANGES_REQUESTED** [review]
Looks good. Just a single test: `test_remove_columns_with_missing_values` needs the change.

**Review by Marsmaennchen221 — CHANGES_REQUESTED** [review]
Looks good so far.

You mised a few files:
_table/test_init.py
_table/test_number_of_columns.py
_table/test_number_of_rows.py
_table/test_remove_columns_with_missing_values.py

**Review by patrikguempel — APPROVED** [review]

**Review by Marsmaennchen221 — DISMISSED** [review]
[LGTM](https://lgtm.party/p/RR17QLA/)

File: tests/safeds/data/tabular/containers/_table/test_init.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
     ("table", "expected"),
     [
         (Table(), Schema({})),
-        (Table({}), Schema({})),
+        (Table(), Schema({})),
```
Comment (zzril) [inline tests/safeds/data/tabular/containers/_table/test_init.py:11]: The IDs suggest that this test is indeed supposed to distinguish between both cases. (Or rather, verify that both create the same table.)

Comment (zzril) [inline tests/safeds/data/tabular/containers/_table/test_init.py:11]: The IDs suggest that this test is indeed supposed to distinguish between both cases. (Or rather, verify that both create the same table.)

**Comment by zzril** [comment]
May also consider doing the same for `Row()` vs `Row({})`.

But can also leave that for another micro-issue. :D

File: tests/safeds/data/tabular/containers/_table/test_init.py (hunk 1 above)
Comment (Marsmaennchen221) [inline tests/safeds/data/tabular/containers/_table/test_init.py:11]:
```suggestion
        (Table({}), Schema({})),
```
@zzril Good point 👍

**Review by zzril — APPROVED** [review]

**Pull request merged**